subsys/bluetooth/host/buf.c

```diff
@@ -141,3 +141,36 @@ struct net_buf *bt_buf_get_evt(uint8_t evt, bool discardable,
 		return bt_buf_get_rx(BT_BUF_EVT, timeout);
 	}
 }
+
```

Comment:
nothing wrong on this line but cannot place comment on commit message.

Commit message says:
> host: Add/Copy mocks for buf.c

are we copying mocks from another location, or how is the `copy` supposed to be understood.
I don't hope we a duplicating code with this commit.

If you are copying files, could you please be a bit more specific and reference the origin of the file in the commit message.

Reply:
No, there is no any duplication here, may be the message is misleading.
I will edit the commit message to make it clear